.github/workflows/e2e-test.yml

```diff
@@ -28,11 +27,14 @@ on:
       - main
       - dev
 
+env:
+  DEFAULT_PYTHON_VERSION: "3.10"
+  EOL_PYTHON_VERSION: "3.9"
```

Comment:
It seems 3.9 is not officially EOL now. Is it okay to leave it as it is?

Reply:
Good call! I just pushed up a commit with the adjusted versions; this probably needs to be applied to the other Python repos too 👍